core/drivers/crypto/hisilicon/sub.mk

```diff
@@ -0,0 +1,5 @@
+cppflags$(sm) += -Icore/drivers/crypto/hisilicon/sec
+cppflags$(sm) += -Icore/drivers/crypto/hisilicon/qm
+
+subdirs-y += sec
+subdirs-y += qm
```

Comment:
Not sure it is worth having two sub-directories. How about `core/drivers/crypto/hisilicon/qm.c` and  `core/drivers/crypto/hisilicon/sec.c` in the previous commits?